xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/ledger/TicketObject.java

```diff
@@ -110,4 +110,11 @@ default Flags flags() {
    */
   @JsonProperty("TicketSequence")
   UnsignedInteger ticketSequence();
+
+  /**
+   * The unique ID of the {@link TicketObject}.
+   *
+   * @return A {@link Hash256}.
+   */
+  Hash256 index();
```

Comment:
Thinking a bit more - should this be pushed up into `LedgerObject`?

Reply:
I'm always a bit hesitant to push fields up to a super class in xrpl4j. I _think_ all ledger object responses have this field, but the code in rippled is unclear and who knows if all future ledger objects will have this field in RPC responses. If it turns out one particular `LedgerObject` has an optional `index` field, or it doesn't have an `index` field and we push `index` up to `LedgerObject`, we'd have to break every single `LedgerObject`, whereas if we copy this field in each `LedgerObject` subclass, the damage is limited to just one object. wdyt?

Reply:
Yeah, probably the right way to go -- I suppose for now this is just an implementation detail, and does leave room for something different (as you point out), as long as we don't omit this field by accident on some `LedgerObject` that should have it.